src/Microsoft.AspNet.Razor/CodeGenerators/CSharpCodeWriter.cs

```diff
@@ -106,11 +106,9 @@ public CSharpCodeWriter WriteStartNewObject(string typeName)
 
         public CSharpCodeWriter WriteLocationTaggedString(LocationTagged<string> value)
         {
-            WriteStartMethodInvocation("Tuple.Create");
             WriteStringLiteral(value.Value);
             WriteParameterSeparator();
-            Write(value.Location.AbsoluteIndex.ToString(CultureInfo.CurrentCulture));
-            WriteEndMethodInvocation(false);
+            Write(value.Location.AbsoluteIndex.ToString(CultureInfo.InvariantCulture));
```

Comment:
Either this change has no user impact and isn't worth making or there's a scenario where the user's culture could result in compilation failures and we should be tracking it as an issue, making sure it the problem doesn't exist elsewhere, testing, ...